### DISCUSSION TITLE
osu! ignores all of my activities

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#22838] [Q&A] [answered]
Hi, i've realized that my osu! profile ALWAYS shows me offline, unless I open osu!'s website.
and nothing of my scores are calculating in it.

![image](https://user-images.githubusercontent.com/89677246/224507690-4b461db2-9a61-490d-b769-854e1a28a2c0.png)
Ranked Score is 0, and Play Count never changes. Even after I played around 20 beatmaps today, this number never changed... .

![image](https://user-images.githubusercontent.com/89677246/224507691-36800c51-52f9-48fb-ba8b-238821991bab.png)
Not any A Scores I got are counted here. (I got some S 's too)

Why this is happening? (my profile https://lazer.ppy.sh/users/32706897 )

**Comment by rvnexe** [comment]
Edit:  Play Count now became 538 somehow (i played many another games after taking this screenshot)

**Comment by ItsShamed** [comment]
You already commented on #22702 that you had the same issue. Please avoid making duplicates. Replying to the relevant post will automatically bump it in the search results and is just enough to voice your concerns.

**Reply by rvnexe** [reply]
Sorry, but I posted this for more proofs of my issue, so developers may understand it better.

**Reply by ItsShamed** [reply]
You can do it in the replies, and I'm pretty sure that the issue is understandable enough with what has been brought so far.

**Reply by rvnexe** [reply]
I'm sorry about all of this. Okay? So stop commenting it.

**Comment by peppy** [comment, marked as answer]
Ranked score and grade totals are not yet implemented for lazer.
